package/ra2mdlauncher.cmd

```diff
@@ -0,0 +1,54 @@
+@echo off
+
+::  This program is free software: you can redistribute it and/or modify
+::  it under the terms of the GNU General Public License as published by
+::  the Free Software Foundation, either version 3 of the License, or
+::  (at your option) any later version.
+::
+::  This program is distributed in the hope that it will be useful,
+::  but WITHOUT ANY WARRANTY; without even the implied warranty of
+::  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+::  GNU General Public License for more details.
+::
+::  You should have received a copy of the GNU General Public License
+::  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+::  This script is a launcher for Yuri's Revenge.
+::  It checks if the gamemd.exe exists and its size.
+::  If the size is 5 MiB or larger, it launches gamemd.exe.
+::  If not, it checks for RA2MD.exe and launches it if found.
+::  If not found, it launches gamemd.exe.
+
+setlocal
+
+set "appPath=%~dp0"
+set "gameMdPath=%appPath%gamemd.exe"
+set "ra2MdPath=%appPath%RA2MD.exe"
+set "minSize=5242880" :: 5 MiB in bytes
```

Comment:
Also, I can now confirm the legit version from a retail CD has the filename `Ra2md.exe` where only `R` is capital. For an Origin or FD copy I don't have access right now.

Reply:
Let's check for every possible case then.

I assumed the size wouldn't be smaller, so I took the exact size of Steam's RA2MD.exe. However, I prefer an exact byte number, and 5 MiB is conveniently just smaller than 5,286,208 bytes.

Reply:
The original 5000000 was selected because it was the median between a gamemd without HD icon and a gamemd with an HD icon. We should tolerate possible reduction of the file size so 5000000 is a good separator.